src/__internal__/focus-trap/focus-trap.spec.tsx

```diff
@@ -798,4 +802,33 @@ describe("FocusTrap", () => {
       expect(screen.getByText(BUTTON_THREE)).toHaveFocus();
     });
   });
+
+  describe("when multiple focus traps are open at once", () => {
+    it("focus moves correctly between the elements of the currently-focused trap", () => {
+      render(
+        <>
+          <MockComponent dataTestId={WRAPPER_ID}>
+            <button type="button">{BUTTON_ONE}</button>
+            <button type="button">{BUTTON_TWO}</button>
+          </MockComponent>
+          <MockComponent dataTestId={SECOND_WRAPPER_ID}>
+            <button type="button">{BUTTON_THREE}</button>
+            <button type="button">{BUTTON_FOUR}</button>
+          </MockComponent>
+        </>
+      );
+
+      focusElement(screen.getByText(BUTTON_ONE));
+      tabPress(WRAPPER_ID);
```

Comment:
yeah I know this will work because I set the default but I think it reads better here being more explicit. I'm not that bothered about it but I can at least explain my reasons.